backends/platform/sdl/psp2/libvita2d/include/bin_packing_2d.h

```diff
@@ -0,0 +1,37 @@
+#ifndef BIN_PACKING_2D_H
```

Comment:
This is part of the shader support. It consists of two libs:
- frangarCJ's libvita2d "fbo" branch with shader support https://github.com/frangarcj/vita2dlib/tree/fbo/libvita2d , 

- frangarCJ's Vita-Shader-Collection release https://github.com/frangarcj/vita-shader-collection/releases/tag/gtu-0.1-v74

Both of these are not part of the VitaSDK. The vita2d "fbo" branch with shader support would overwrite the standard SDK version of libvita2d on "make install" and break compatibility with other compilations. The vita-shader-collection doesn't even have a "make install."

The shaders can be compiled manually into .o files, see makefile here: 
https://github.com/frangarcj/vita-shader-collection

but it requires a full qemu setup which I could not setup in a few hours to get it to work.

Am I allowed to include .a libraries in scummVM? Or shall I move all of this library stuff out of the branch and just put a note in the readme-psp2.md how to install it to compile?